instrumentation/grpc/lib/opentelemetry/instrumentation/grpc/patches/rpc_desc.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module GRPC
+      module Patches
+        # Patches to prepend to ::GRPC::RpcDesc.
+        module RpcDesc
+          # This is the entry point to all server-side calls.
+          def run_server_method(active_call, mth, inter_ctx = ::GRPC::InterceptionContext.new)
+            attrs = {
+              'rpc.system' => 'grpc',
+              'rpc.service' => mth.owner.service_name,
+              'rpc.method' => mth.name.to_s
```

Comment:
small nit (was working on something related and figured i'd see how this was implemented, havent done a full review)

I think semantic conventions require either `net.peer.ip` _or_ `net.peer.name`: https://github.com/open-telemetry/opentelemetry-specification/blob/87a5ed7f0d4c403e2b336f275ce3e7fd66a8041b/specification/metrics/semantic_conventions/rpc.md#attributes

Reply:
Ah, you're right. It's even on the TODO list for the PR and I neglected to do it. 😆